README.md

```diff
@@ -218,12 +280,11 @@ As an autonomous experiment, Auto-GPT may generate content or take actions that
 
 By using Auto-GPT, you agree to indemnify, defend, and hold harmless the developers, contributors, and any affiliated parties from and against any and all claims, damages, losses, liabilities, costs, and expenses (including reasonable attorneys' fees) arising from your use of this software or your violation of these terms.
 
-## 🐦 Connect with Us on Twitter 
```

Comment:
The trailing space fix is both unrelated and in #590

Reply:
Please comment on https://github.com/Torantulino/Auto-GPT/pull/142